src/main/java/com/numaproj/pulsar/SpringPulsarApplication.java

```diff
@@ -0,0 +1,15 @@
+package com.numaproj.pulsar;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.pulsar.annotation.EnablePulsar;
+
+@SpringBootApplication
+@EnablePulsar
```

Comment:
interesting annotation, I would like to learn more about it. Let's sync offline on this